_includes/install_command.html

```diff
@@ -0,0 +1,28 @@
+{%- capture code -%}
+brew {{include.modifier}} install {{include.item}}
+{%- endcapture -%}
+
+<style>
+    #copyButton {
+        background: transparent;
+        border: none;
+        visibility: hidden;
+    }
+</style>
+
+<code>{{code}}</code>
+<button
+    id="copyButton"
+    aria-label="Copy to clipboard"
+    onclick="navigator.clipboard.writeText('{{code}}')"
+>
+    📋
+</button>
+
+<script>
+    if (navigator.clipboard) {
+        document
+            .getElementById("copyButton")
+            .style.removeProperty("visibility");
```

Comment:
```suggestion
        document.getElementById("copyButton")
                .style.removeProperty("visibility");
```

Also, this didn't seem to work for me in my local testing. Perhaps this should be a function called by an `onload`?

Reply:
Yeah, that was another issue. It only removes the style defined on the element and not in CSS.
This has been fixed in the latest update.